### PR TITLE
29 p108 orwtpd1 getedats

### PR DESCRIPTION
**Pull request by kekuso**
- implement model and tests for p108 ORWTPD1 GETEDATS

### CODE REVIEW

File: prototypes/parameters/p108_orwtpd1Getedats.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        processRPCInvocation(rpcArgs) {
+            // An error will be thrown if any arg is passed into function
+            paramErrorHandler(rpcArgs);
+
+            // Get the User ID
+            const user = parameterService.convertFileToEntity(VDM.userId());
+            const userId = VDM.userId().split('-')[1];
+
+            // Get the user's service and section
+            const value = VDM.describe(`200-${userId}`);
```
Comment (mfuroyama) [inline prototypes/parameters/p108_orwtpd1Getedats.js]: You can replace the 2 lines above with:
```Javascript
const value = VDM.describe(VDM.userId());
```

File: prototypes/parameters/p108_orwtpd1Getedats.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+const VDM = require('../../../VDM/prototypes/vdm');
+const paramErrorHandler = require('../noArgsErrorHandler');
+const parameterService = require('../../../VDM/prototypes/parameterService');
+
+const RPC_LOCKER_MODEL = [
+    {
+        id: 'ORWTPD1_GETEDATS',
+        name: 'ORWTPD1 GETEDATS',
+        function: 'GETEDATS^ORWTPD1',
+        parameters: ['ORQQEAPT ENC APPT', 'ORQQEAPT ENC APPT STOP'],
```
Comment (mfuroyama) [inline prototypes/parameters/p108_orwtpd1Getedats.js]: The first parameter label, `'ORQQEAPT ENC APPT'`

I think should be:

`'ORQQEAPT ENC APPT START'`

Comment (kekuso) [inline prototypes/parameters/p108_orwtpd1Getedats.js]: good catch!

File: prototypes/parameters/p108_orwtpd1Getedats.js (hunk 1 above)
Comment (kekuso) [inline prototypes/parameters/p108_orwtpd1Getedats.js]: nice, thanks

File: prototypes/parameters/p108_orwtpd1Getedats.js [hunk 3]
```diff
@@ -24,10 +24,9 @@ const RPC_LOCKER_MODEL = [
 
             // Get the User ID
             const user = parameterService.convertFileToEntity(VDM.userId());
-            const userId = VDM.userId().split('-')[1];
 
             // Get the user's service and section
-            const value = VDM.describe(`200-${userId}`);
+            const value = VDM.describe(VDM.userId());
```
Comment (mfuroyama) [inline prototypes/parameters/p108_orwtpd1Getedats.js]: Yeah...sorry about that!  My bad on this one...I had non-optimal code in the example wiki!  The page is updated now...it's a process!  😄

**Pull request merged**